package.json

```diff
@@ -45,6 +46,7 @@
     "ga-gtag": "^1.0.1",
     "handlebars": "^4.7.2",
     "handlebars-loader": "^1.7.1",
+    "highlight.js": "^10.7.2",
     "html-webpack-plugin": "^4.0.0-beta.8",
     "jquery": "^3.4.1",
     "jquery-caret-plugin": "^1.5.2",
```

Comment:
Can you move the new dependencies to an independent first commit?  That way we can merge quickly the main thing that might merge conflict.

Also, do these belong in `devDependencies`, given that this is a page only used in a development environment?  

(It's possible that `webpack` considerations limit our ability to do that).